### ISSUE TITLE
TLS certificate check fails

### ISSUE THREAD

**Issue by twiddern** [madcowfred/GoPostStuff#13]
Hi,

the check for a valid TLS certificate fails

2014/11/30 16:18:59  INFO      [FARMAMS] Starting 20 connections
2014/11/30 16:18:59  CRITICAL  [FARMAMS] Error while connecting: tls: either ServerName or InsecureSkipVerify must be specified in the tls.Config
2014/11/30 16:18:59  CRITICAL  [FARMAMS] Error while connecting: tls: either ServerName or InsecureSkipVerify must be specified in the tls.Config

So I had a look into the https://github.com/madcowfred/GoPostStuff/blob/master/simplenntp/simplenntp.go but there's only the InsecureSkipVerify mentioned. ServerName is never mentioned to test. http://golang.org/pkg/crypto/tls/#Conn

**Comment by madcowfred** [comment]
Add 'InsecureSSL=on' to the [server] block in your config?

**Comment by twiddern** [comment]
Yup, but this should be only a workaround. It would be much better to validate the certificate

**Comment by madcowfred** [comment]
Is the certificate actually valid? I didn't run into any issues testing with errrr Newshosting whenever I was testing TLS.

**Comment by tYGjQCsvVI** [comment]
@madcowfred Yes, it fails, yes the SSL cert is valid.

EDIT:- For reference, here's my configuration:-
https://paste.debian.net/plainh/5a5c0c19

You don't need valid credentials to hit the error (As obviously authentication is done above the encryption layer, which is never established).

**Comment by twiddern** [comment]
Yup I totally forgot this issue
